app/workers/Indexer.scala

```diff
@@ -68,7 +70,17 @@ object Indexer {
     // also add all topics
     dataMap += "topicSchemeTag" -> toJson(item.topics.map(_.scheme.get.tag))
     dataMap += "topicTag" -> toJson(item.topics.map(_.tag))
-    WS.url(indexSvc + "item/" + item.id).put(stringify(toJson(dataMap)))
+    println("Item index: " + dataMap)
+    println(indexSvc + "item/" + item.id)
+
+    if (indexSvc.contains("bonsai.io")) {
```

Comment:
We may want a global switch for this rather than specifically checking for a string in the `indexSvc` url. Something like `hub.index.basic_auth` which defaults to false but can be set true via ENV so the logic wherever we need it would be to check that value and if true, use the auth code if false, use the non-auth code.